src/FormBuilder/ViewModels/FormBuilderViewModel.php

```diff
@@ -19,6 +19,7 @@ public function storageData(int $donationFormId): array
         $donationForm = DonationForm::find($donationFormId);
 
         return [
+            'formPageURL' => add_query_arg(['p' => $donationFormId], site_url()),
```

Comment:
This uses the `?p={id}` query parameter so that it always works, regardless of the page slug setting (which may not be unique when changed, that is, until the server can update it).